### PR TITLE
[RHACS] Clarify default policy options

### PR DESCRIPTION
**Pull request by kcarmichael08**
Versions: 

- merge to: `rhacs-docs`
- cherry pick to: `rhacs-docs-3.70`

- Labels: `RHACS`
- Issue: none (raised via Slack)
- [Preview](https://openshift-docs-qo94r6neb-kcarmichael08.vercel.app/openshift-acs/master/operating/manage-security-policies.html)

### CODE REVIEW

**Review by md2119 — APPROVED** [review]
LG

**Comment by md2119** [comment]
Kerry, this default policy changes do not need to be backported to 69 and 68

File: modules/use-default-security-policies.adoc [hunk 1]
```diff
@@ -34,6 +34,11 @@ The default policies have preconfigured parameters and belong to categories such
 * System Modification
 * Vulnerability Management
 
-You can edit these categories create your own categories.
+You can edit these categories and create your own categories.
```
Comment (mburke5678) [inline modules/use-default-security-policies.adoc:37]: This is this still not clear to me. Is it
You can edit these categories _**to**_ create your own categories.
You can edit these categories _**or**_ create your own categories.

**Comment by mburke5678** [comment]
@kcarmichael08 One tiny nit. Otherwise LGTM

**Pull request merged**